sass/_cta.scss

```diff
@@ -0,0 +1,13 @@
+.cta__wrapper {
+  @include component-box;
+  position: fixed;
+  top: 50%;
+  transform: translateY(-50%);
+  right: 0;
+  padding: 0;
+  z-index: 9999;
+  width: 3.75rem;
```

Comment:
It made sense to me to have width on the child elements. You'd set the child elements to have the same width and height to make squares, and it was good to see height and width next to each other with the same value. So then the parent container just fit the children and didn't need the dimensions defined.